R/create.autoencoder.R

```diff
@@ -8,12 +8,12 @@ create.autoencoder <- function(
 	) {
 
 	# input checks
-	if (class(data.matrix) != 'matrix') {
-		stop('data.matrix needs to be a matrix');
+	if ('matrix' != class(data.matrix) && 'data.frame' != class(data.matrix)) {
+		stop('data.matrix needs to be a matrix or a data.frame');
```

Comment:
For consistency, consider the following:

```suggestion
		stop('data.matrix must be a matrix or a data frame');
```